### ISSUE TITLE
Upgrade Problem umsetzen der jarvis-DPs auf v3.2.0 besteht weiterhin

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#2803]
Auch mit `v3.2.0-rc.5` ist das Problem noch da.
CSS wird nicht korrekt umgesetzt.
https://forum.iobroker.net/topic/68566/jarvis-v3-2-x-just-another-remarkable-vis/323 und ff

**Comment by Tom54497** [comment]
habe das gleiche Problem

**Comment by Zefau** [comment]
@Tom54497 hast du auf `rc-5` versucht ein Backup wiederzuherstellen?

**Comment by Tom54497** [comment]
> @Tom54497 hast du auf `rc-5` versucht ein Backup wiederzuherstellen?

nein

**Comment by Zefau** [comment]
Versuch das mal bitte

**Comment by Tom54497** [comment]
Nun funktioniert es.

Update auf v3.2.0-rc.5

1. Datenpunkt Wert in CSS setzen:
{
  "version": 3.2,
  "signature": 1234,
  "styles": ""
}

2. Datenpunkt Wert in SCRIPTS setzen:
{
  "version": 3.2,
  "signature": 3338908027751811,
  "scripts": ""
}

3. Browser Cache löschen

**Comment by Erwin40** [comment]
Wenn es über Backup restore funktioniert, kann das hier doch geschlossen werden?
Ich weiß ich nerve, aber für mein Gefühl hat der Adapter einfach zuviele Issues.
So wird da nie eine Final Version fertig ;-)

**Comment by Zefau** [comment]
> Ich weiß ich nerve, aber für mein Gefühl hat der Adapter einfach zuviele Issues.

Jo, sind zu viele Issues, wobei es ja auch viele Feature Requests sind. Die Bugs müssen wir aber nachhaltig beheben, sonst gibt es davon ja immer mehr ;-)

**Comment by mcuiobroker** [comment]
Auch dies Problem ist nicht behoben?

**Comment by Zefau** [comment]
Dann sag mir, wie ich es nachstelle?

**Comment by mcuiobroker** [comment]
Nachstellen kannst du es sehr wahrscheinlich nicht, aber du kannst eine Prüfung einbauen, ob die jarvis-ORIG-DPs den korrekten Aufbau haben, nachdem die Umstellung / Update gelaufen ist. Denn dies ist ja für die Meisten das Hauptproblem.
In den jarvis-DPs scripts und CSS wird ja die signature nicht umgesetzt und daraus folgt das Problem z.B. permaneter Reload.

**Comment by Zefau** [comment]
Die Version `v3.2.0-rc.5` sollte den Fehler beheben - bitte einmal prüfen. Sofern es behoben ist, gerne das Issue schließen.<br /><hr />The version `v3.2.0-rc.5` should fix the bug - please verify. If the bug has been solved, you may close the issue.
